src/store/store.js

```diff
@@ -60,9 +65,43 @@ export default {
     setAssistanceItems: function(state, {index, result}) {
       state.assistanceItems[index] = result;
     },
-    setConfig: function(state) {
+    setConfig: function(state, result) {
+      if (result.hasOwnProperty('theme')) {
+        state.settings.theme = result['theme'];
+      }
+
+      // Conformance. Must be 'light' or 'dark'. Default is light
+      if (state.settings.theme == null || state.settings.theme === '') {
+        state.settings.theme = 'light';
+      }
+      document.getElementsByTagName('HTML')[0]
+          .setAttribute('class', state.settings.theme);
+
+      if (result.hasOwnProperty('zoom')) {
+        state.settings.zoom = result['zoom'];
+      }
+      // Conformance. Assume here the number is not zero and in range.
+      if (state.settings.zoom == null || state.settings.zoom == 0.0 ||
+          typeof state.settings.zoom !== 'number') {
+        state.settings.zoom = 1.0;
+      }
+      const wf = require('electron').webFrame;
+      wf.setZoomFactor(state.settings.zoom);
+
       state.configLoaded = true;
     },
+    setZoom: function(state, factor) {
+      state.settings.zoom = factor;
+      const wf = require('electron').webFrame;
+      wf.setZoomFactor(factor);
+
+      updateConfiguration(remote, state.settings);
+    },
+    setTheme: function(state, theme) {
+      state.settings.theme = theme;
+      document.getElementsByTagName('HTML')[0].setAttribute('class', theme);
```

Comment:
This is a bit more complicated then it needs to be
```suggestion
      document.documentElement.className = theme;
```